src/fides/api/cryptography/cryptographic_util.py

```diff
@@ -16,12 +17,26 @@ def decode_password(password: str) -> str:
         return password
 
 
-def hash_with_salt(text: bytes, salt: bytes) -> str:
-    """Hashes the text using SHA-512 with the provided salt and returns the hex string
-    representation"""
+def hash_credential_with_salt(text: bytes, salt: bytes) -> str:
+    """
+    Hashes the text using bcrypt with the provided salt and returns the hex string representation.
+
+    **This is a computationally expensive hash that should only be used to hash passwords or other
+    credential information. For general data hashing use `hash_value_with_salt`.**
+    """
     return bcrypt.hashpw(text, salt).hex()
 
 
+def hash_value_with_salt(text: bytes, salt: bytes) -> str:
+    """
+    Hashes the text using SHA-256 with the provided salt and returns the hex string representation.
+
+    **This is a quick hashing function only suitable for general data, use `hash_credential_with_salt` to hash
+    passwords or other credentials.**
+    """
+    return hashlib.sha256(text + salt).hexdigest()
```

Comment:
Splitting this out into two hashing functions, bcrypt will still be used for credentials, but we will use SHA-256 for non-credential values like identities.